admin/spec/features/return_reasons_spec.rb

```diff
@@ -79,7 +79,7 @@
       expect(page).to be_axe_clean
     end
 
-    it "closing the modal keeps query params" do
+    it "closing the modal keeps query params", :flaky do
```

Comment:
I would like this change to be a dedicated commit, because it is unrelated to the actual change in this PR

Reply:
@tvdeyen is this the only thing holding it up?

Reply:
@tvdeyen  I have separated it into a dedicated commit for the flaky test here https://github.com/solidusio/solidus/pull/6094/commits/4b11b0527ae9d642d4bd31c1d96cb20c2b5fc3dd 
please let me know if you need me to remove it or send it in a different PR